src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/TriviaDataFactory.vb

```diff
@@ -161,11 +161,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
             Contract.ThrowIfFalse(space >= -1)
 
             If space >= 0 Then
-                Return GetSpaceTriviaData(space, result.TreatAsElastic)
+                Dim forceSecondTokenIsFirstOnLine =
+                    token1.IsKind(SyntaxKind.EmptyToken) AndAlso
+                    token1.Parent.IsKind(SyntaxKind.OmittedArgument) AndAlso
+                    token2.IsKind(SyntaxKind.CommaToken)
```

Comment:
At a minimum this should check that the EmptyToken *is* actually the first token on the line.